### DISCUSSION TITLE
Retention time tolerance and Kovats Index Extraction for GC-EI

### DISCUSSION THREAD

**Discussion by kriskiland** [mzmine/mzmine#1819] [Q&A]
Hi all, new user (within the past month) of MZmine here. Been super happy with everything in the application so far. I'm primarily doing untargeted GC-EI-Orbi work on volatile organics. Our samples are rich with alkanes, and so retention index calculation is much needed for identification. The Kovats Index Extraction module isn't available in the "Tools" dropdown of my MZmine v4.0.3. The last I saw mentioned of this on the forums was in the Issues section, ~5 months ago ("no Kovats index extraction in Tools #1370"). The response from the developers was that the GUI interface was not operational, but they're working on it. Just wondering if anyone knows if there has been any progress made on getting that module back up and running or an estimated timeline? 

Additionally, I also have a targeted GC-EI-ToF project. For this I think I can get away with using the "retention time tolerance" option in the spectral library search module -- however, I'm not sure how to implement this. Where do I enter the expected retention times for my targets? I assume this would be within my in-house library (the NIST MS Search program), but I don't see any field to enter these values. Alternatively, I guess I could use the "Local database search" module, but this seems to me to be more for soft ionization rather than EI.

Thanks for any/all help! Much appreciated.

**Comment by dav7841** [comment]
Just want to bump this issue here as I also to work with Kovats index for untargeted GCMS analysis and am waiting for a fix on this. 
To answer the above question regarding retention times, libraries created and edited in NIST don't have retention time fields, only retention index. You can however, export the libraries to msp format from the librarian tab in NIST and then open this up in AMDIS. The library editor here  works with retention times as well as indexes. It's even better in my view when building libraries from scratch to use AMDIS as it reads the compound info directly from the NIST searches into your new library. Less tedious than typing/copying and pasting everything in. Only issue with AMDIS msp libs though is the spectrum format needs to modified as the mzmine doesn't read intensity mz pairs in brackets. The brackets will need to be removed and each intensity, mz pair on their own line. A single space in between the mz and intensity value too.
